lib/web-extensions/scrollplugin.c

```diff
@@ -70,6 +70,9 @@ on_document_loaded (WebKitWebPage  *page,
                                               G_CALLBACK (signal_scroll),
                                               FALSE,
                                               data);
+
+  // Always let the client know about the scroll status
+  signal_scroll (WEBKIT_DOM_EVENT_TARGET (document), NULL, data);
```

Comment:
The `WEBKIT_DOM_EVENT_TARGET` cast is not necessary, I think, since the function itself takes a `WebKitDOMDocument*`